.github/pull_request_template.md

```diff
@@ -21,3 +12,21 @@ For more information on the requirements for pull requests, please see [the CONT
     Write good, descriptive Git commit messages.
 
 -->
+
+<!--
+
+For more information on the requirements for pull requests, please see [the CONTRIBUTING.md](CONTRIBUTING.md).
+
+-->
+
+[] I confirm this contribution is my own and I agree to license it with Apache 2.0.
+
+[] I confirm the moderators may change the PR before merging it in.
+
+For new board enablements only.
+
+[] I confirm the board is [Mbed Enabled](https://www.mbed.com/en/about-mbed/mbed-enabled/introduction/) and passes the Mbed Enabled test set.
+
+[] I confirm the contribution has been tested properly and tests results for [TESTS](https://github.com/ARMmbed/mbed-os-example-pelion/tree/master/TESTS) are attached.
+
+[] I confirm `mbed-os.lib` and `mbed-cloud-client.lib` hashes were and/or the content in those folders was not modified in order to pass the tests.
```

Comment:
I confirm I haven't modified the ... hashes or the content in the folders (which folders?) to pass the tests.